emcc.py

```diff
@@ -2015,9 +2030,9 @@ def include_and_export(name):
       elif settings.LINKABLE:
         diagnostics.warning('experimental', '-s LINKABLE + pthreads is experimental')
 
-      default_setting('SUPPORT_LONGJMP', 0)
-      if settings.SUPPORT_LONGJMP:
-        exit_with_error('SUPPORT_LONGJMP is not compatible with pthreads + dynamic linking')
+      default_setting('SUPPORT_LONGJMP', 'none')
+      if settings.SUPPORT_LONGJMP == 'emscripten':
+        exit_with_error('SUPPORT_LONGJMP=emscripten is not compatible with pthreads + dynamic linking')
```

Comment:
Maybe leave these two lines as is?  (Since I imagine `SUPPORT_LONGJMP` of any kind is currently not support .. or at least not tested with pthreads + dynamic linking).

Reply:
Is there a reason Wasm SjLj is not supported with pthreads + dynamic linking? Dynamic linking should be no problem because we support it the same way we did for Wasm EH, and pthreads is not tested yet, but it's not like "that's an unsupported combination"..?